guides/Retrospectives.md

```diff
@@ -0,0 +1,70 @@
+
+## What is a retrospective?
+A retrospective is a meeting held typically at the end of a sprint that gives the team the opportunity to reflect on what happened in the sprint and identify actions for improvement moving forward.
+
+## When to run retrospectives?
+Generally, a retrospective is held at the end of a sprint. Holding it at the end of a sprint is suggested because the team has completed a sizeable amount of work that they can reflect on. It also allows the team to quickly implement ideas and improvements in the next sprint (that can then be followed up on and discussed in the next retro). The most important thing regarding when retros are held is to have them frequently enough to allow for iteration and continuous improvement.
+
+## Who should join a retrospective?
+Every team member (such as a PM, a designer, developers, and an analyst when necessary) is encouraged to join a retrospective. Also, it may be valuable to include other stakeholders when a notable milestone is completed. For example, after the launch of the Year In Art project it might be helpful to include an editorial team member for that retrospective. It is also important to be mindful of the meeting size so as to keep the focus of that retrospective manageable.
+
+## How to run retrospectives?
+
+__Facilitator__
+A retro needs a facilitator whose role is to:
+
+- Manage time.
+- Assist the group in ordering/organizing items.
+- Engage and encourage discussion (by asking questions/reviewing items).
+- Take notes of action items.
+
+
+Many articles, blog posts, and books suggest having a dedicated facilitator rather than having one of the team members play this role, which would help remove bias. In a small team, that may not be an option but is something for the facilitator to keep in mind.
+
+
+__Format__
+There are dozens of retrospective formats in Agile [Retrospectives](https://www.amazon.com/Agile-Retrospectives-Making-Teams-Great/dp/0977616649) or [online](http://retrospectivewiki.org/index.php?title=Retrospective_Plans). The most commonly used one is called “Glad, Sad, Mad.” According to Retrospectivewiki.org, this format is described as follows:
+
+
+ >Issues, changes or observations made during a sprint are listed by all participants and then categorized as either Glad, Sad or Mad. These broadly represent positive notes about the sprint/team, negative notes about the sprint/team, and “gripes” (which are often more broad than just actions of the team), respectively. Participants then vote on the issues to be discussed and the issues are then discussed from highest to lowest number of votes until all issues have been discussed or there is no time remaining.
+
```

Comment:
Interesting! I'm more familiar with "Yay / Meh / Nay" instead of "Glad, Sad, Mad." I wonder how these prescriptive emotions dictate a difference in discussion. (ie: I wonder if "Mad" create more angry responses as opposed to more constructive articulation with struggle?). For — Yay/Meh/Nay, its looser; have found "Meh" to actually be a really helpful column because a lot of ambiguous feelings are captured here that lead to interesting discussions / uncover deeper problems (lol).